python/lib/sift_grafana/sift_query_model.py

```diff
@@ -0,0 +1,56 @@
+from typing import List, Optional
+
+from pydantic import BaseModel, Field, model_validator
+from typing_extensions import Self
+
+
+# These are the models that are used to validate the input data for the Sift Grafana Plugin queries
```

Comment:
Are these internal doc-comments or are they meant for the public? If they're meant for users, could you format it like in this [example](https://github.com/sift-stack/sift/blob/735ddca3ea26e22142c29518b5c32d819b968c5a/python/lib/sift_py/ingestion/service.py#L21)?

If just internal, feel free to just resolve this comment.

Reply:
Yeah that makes sense